snippets/asn.xml

```diff
@@ -1,36 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
 <snippets>
-<snippet group="ASN.1" trigger="sequence" id="sample_snippet_definition">Sample DEFINITIONS ::= BEGIN
-
-    $name$ ::= SEQUENCE {
-        trackingNumber INTEGER,
-        question       IA5String
-    }
-
-    Test ::= SEQUENCE {
-        other $name$
-    }
-
-END
+<snippet group="ASN.1" trigger="sequence" id="sequence">SEQUENCE
```

Comment:
I'm not sure, but I had some errors while using the same  id across different snippets files/groups. So ID might need to be globally unique. Needs verification. If so, some prefix like `Asn1Acn.Asn1.` could be added

Reply:
This is true, I have tested and duplicated ids might not be working. I will follow prefixing rule shared among other snippet files, which is using underscores.